lib/storage/src/content_manager/toc.rs

```diff
@@ -103,6 +103,10 @@ impl TableOfContent {
         create_dir_all(&snapshots_path).expect("Can't create Snapshots directory");
         let collections_path = Path::new(&storage_config.storage_path).join(COLLECTIONS_DIR);
         create_dir_all(&collections_path).expect("Can't create Collections directory");
+        if storage_config.storage_path != storage_config.tmp_path {
+            let tmp_path = Path::new(&storage_config.tmp_path.clone()).to_owned();
+            create_dir_all(tmp_path).expect("Can't create temporary files directory");
+        }
```

Comment:
Though this is _fine_ now, we better try to create the directory anyway. This would break if we use the temporary directory in other places as well.

It will simply do nothing if the directory already exists:

```suggestion
        if let Some(path) = storage_config.tmp_path {
            create_dir_all(path).expect("Can't create temporary files directory");
        }
```